utils/kvrocks2redis/kvrocks2redis.conf

```diff
@@ -23,7 +29,7 @@ kvrocks 127.0.0.1 6666
 
 # Enable cluster mode.
 #
-# Default: yes
+# Default: no
 cluster-enable yes
```

Comment:
Is there a typo?

Reply:
@PragmaTwice You mean the default is `no` but the actual value in the config file is `yes`? I was thinking about the change. Should I change it to `no`?

Reply:
Yeah but I am not familiar with these components. cc @git-hulk

Reply:
This configuration should be meanless to kvrocks2redis

Reply:
Yes, the default value should be `no`

Reply:
Changed.